scripts/build_test.py

```diff
@@ -14,29 +14,313 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import StringIO
+import collections
+import os
+import random
+import re
+import subprocess
+import threading
+
 # pylint: disable=relative-import
 import build
 from core.tests import test_utils
 
 # pylint: enable=relative-import
 
+TEST_DIRECTORY = os.path.join('core', 'tests', 'build', '')
```

Comment:
TEST_ROOT_DIRECTORY

(Or maybe end this and the next one with _DIR to match the others.)